libafl_frida/src/executor.rs

```diff
@@ -149,7 +149,7 @@ where
         }
 
         #[cfg(windows)]
-        initialize(&gum);
+        initialize(&gum).unwrap();
```

Comment:
We shouldn't unwrap here either, but return a Result, unless if we are confident this cannot fail (which may be the case?)

Reply:
okay, in case we've decided not to panic and we want handle the error by passing it on upper level, how could we do that in a constructor? The only value we can return is Self.

Reply:
There are no constrictors in rust, it's a normal method returning `Self` right now. You'd change the method signature to return `Result<Self, Error>` instead ;)

Reply:
Thanks. I suspect you're not going to be happy if `FridaInProcessExecutor::new()` returns `Result<Self, Error>`, right?

Reply:
I'd be very happy about it, if that's the right thing to do here :)
Basically only panic inside the library (and/or unwrap) if something will never fail. Finding functions in windows may be one such example, I'll leave that decision up to you

Reply:
Okay think panic is more suitable here